### PR TITLE
Check the benchmarks in the CI

### LINKED ISSUE #209: Cargo check of benchmark in CI

### PR DESCRIPTION
**Pull request by Kerollmops**
Fixes #209.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@ jobs:
         with:
           command: test
 
+      - uses: actions-rs/cargo@v1
+        with:
+          command: bench
+          args: --no-run -p benchmarks
+
```
Comment (ManyTheFish) [inline .github/workflows/test.yml:44]: can't we modify the cargo build action L32 by adding `args` under the `command` in order to check all packages?
```yaml
      - uses: actions-rs/cargo@v1
        with:
              command: build
              args: # ...
```

Comment (Kerollmops) [reply .github/workflows/test.yml:44]: Yup we could but add this too!

Comment (Kerollmops) [reply .github/workflows/test.yml:49]: I just added a `check --all` CI job.

Comment (ManyTheFish) [reply .github/workflows/test.yml:49]: If we already have a `cargo check --all` what is the purpose of an additional `cargo bench --no-run`?

Comment (Kerollmops) [reply .github/workflows/test.yml:49]: That's because `cargo bench` compiles the benchmarks which `cargo check` don't.

**Review by ManyTheFish — APPROVED** [review]

**Comment by Kerollmops** [comment]
Thank you @ManyTheFish!
bors merge

**Pull request merged**